packages/amplication-build-manager/src/build-logger/build-logger.controller.ts

```diff
@@ -21,7 +21,7 @@ export class BuildLoggerController {
       value: logEntry,
     };
     await this.producerService.emitMessage(
-      this.configService.get(Env.DSG_LOG_TOPIC),
+      this.configService.get(KAFKA_TOPICS.DSG_LOG_TOPIC),
```

Comment:
The topic name is now hardcoded in an enum and not loaded anymore from environment variables
```suggestion
      KAFKA_TOPICS,
```